### PR TITLE
Advance current development environment to Fedora 40

### PR DESCRIPTION
**Pull request by bgurney-rh**
Related: stratis-storage/project#706

### CODE REVIEW

**Comment by mulkieran** [comment]
@bgurney-rh You can allow the too-many-arguments lint.

File: src/hs_dbus_signature/_signature.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 _CODES = ("b", "d", "g", "h", "i", "n", "o", "q", "s", "t", "u", "v", "x", "y")
 
 
+# pylint: disable=too-many-arguments
 def dbus_signatures(
```
Comment (mulkieran) [inline src/hs_dbus_signature/_signature.py]: Please move the ```disable``` line onto the line below. The ```disable``` above the function call disables that lint for everything below it in the file, including all additional functions in the file (in this case, this is the only function, but that's unusual).

```def dbus_signatures(   #pylint...```

**Review by mulkieran — CHANGES_REQUESTED** [review]
One minor nit...

File: src/hs_dbus_signature/_signature.py (hunk 1 above)
Comment (bgurney-rh) [reply src/hs_dbus_signature/_signature.py]: OK; amending that last commit with the fix.

**Review by mulkieran — APPROVED** [review]

**Pull request merged**